src/pages/home/live-pricing/_types.ts

```diff
@@ -0,0 +1,30 @@
+export type TMarketData = {
+    ask: number
+    bid: number
+    day_percentage_change: number
+    market: string
+    market_ranking: string
+    spread: number
+    symbol: string
+}
+
+export type TMarketDataResponse = {
+    trading_platform_asset_listing: any
```

Comment:
Any chance to provide what we expects here? We can log it and write what we getting
```suggestion
    trading_platform_asset_listing:{ mt5: {  assets: TAssets } }
```